doc/modules/ROOT/pages/about/release_policy.adoc

```diff
@@ -1,7 +1,6 @@
 = Release Policy
 
-We're following http://semver.org/[SemVer].footnote:[As much as one can be
-following it when the major version is 0.] At this point bumps of the
+We're following http://semver.org/[SemVer]. At this point bumps of the
```

Comment:
You can also remove much of what follows, as it's not really relevant right now (after the release of 1.0).